invenio_rdm_records/fixtures/records.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Communities fixture module."""
+
+from ..utils import get_or_create_user
+from .fixture import FixtureMixin
+from .tasks import create_demo_record
+
+
+class RecordsFixture(FixtureMixin):
+    """Records fixture."""
+
+    def __init__(self, search_paths, filename, delay=True):
+        """Initialize the record's fixture."""
+        super().__init__(search_paths, filename)
+        self.admin = get_or_create_user("admin@inveniosoftware.org")
+        self.delay = delay
+
+    def create(self, entry):
```

Comment:
Communities, Records and probably other fixtures have all:
- A function that is being called (e.g. `create_demo_record`)
- Some parameters to pass to the function
- The function being delayed or not

I am wondering if we could abstract it in the `FixtureMixin`, something like (pseudocode):

```python
self __init__(..):
    self.create_func = ...
    self.delay = ...

self create(*args, **kwargs):
    if delay:
        self.create_func.delay(*args, **kwargs)
    else:
                self.create_func(*args, **kwargs)
```

Then we would only specify attributes to the class and avoid duplicated code 🔥